log-viewer-cli/bin/logback.xml

```diff
@@ -0,0 +1,36 @@
+<configuration scan="true" debug="false">
```

Comment:
`logback.xml` is a configuration file, not a binary. So it should be in the same folder as `config.conf`. You can either put `logback.xml` to the root, or move `logback.xml` and `config.conf` to "conf" directory.